README.md

```diff
@@ -32,6 +33,19 @@ query = server.query()
 print(f"The server has the following players online: {', '.join(query.players.names)}")
 ```
 
+Bedrock Edition
+```python
+from mcstatus import MinecraftBedrockServer
+
+# If you know the host and port, you may skip this and use MinecraftBedrockServer("example.org", 19132)
+server = MinecraftBedrockServer.lookup("example.org:19132")
+
+# 'status' is the only feature that is supported by Bedrock at this time.
+# In this case status can also include motd, map, gamemode, and players_max. (ex: status.gamemode)
```

Comment:
Fix the wording here, "can also include" is weird because it can include the listed things on top of what?
```suggestion
# In this case status includes motd, map, gamemode, and players_max. (ex: status.gamemode)
```

Reply:
I left it out because I used your examples in the code and did not include them in the comment. I added those to the comment and changed it to include. Great work on this project by the way. This has been very useful to me.